tests/formats_test/selinux_test.py

```diff
@@ -51,3 +52,8 @@ def test_mount_selinux_ext2fs(self):
     def test_mount_selinux_xfs(self):
         """ Test of correct selinux context parameter value when mounting XFS"""
         self.exec_mount_selinux_format(fs.XFS)
+
+    def _clean_up(self):
+        super(SELinuxContextTestCase, self).tearDown()
```

Comment:
Please remove this line. We don't want to call `tearDown` from `_clean_up` method.